WcaOnRails/app/views/delegates_panel/index.html.erb

```diff
@@ -11,4 +11,16 @@
       <%= link_to "Delegates", delegates_stats_path %>
     </li>
   </ul>
+
+  <% if current_user&.can_view_senior_delegate_material? %>
+    <h1>Senior Delegates panel</h2>
+
+    <ul>
+      <li>
+        <%= link_to "WCA Candidate Delegate Nomination", "https://docs.google.com/forms/d/e/1FAIpQLSe0g8nWUTdmbfds_bfdZLJxK_B4oMv68M-I4rj9AvluFH5VJA/viewform?usp=sf_link" %>
```

Comment:
I think it would make more sense to say "WCA Candidate Delegate nomination form" and "WCA Candidate Delegate Promotion"*
Or to say "Forms" above them.

* Alberto asked to include one more form (for Demotion)

Reply:
Ok, I've added "form" to the end of all of these links.